ferris_cast_core/tests/parse_rss_test.rs

```diff
@@ -0,0 +1,49 @@
+use ferris_cast_core::{parse_rss, Episode, Podcast};
+use std::path::PathBuf;
+
+#[test]
+fn is_ok() {
+    let buf = PathBuf::from("./tests/assets/test_rss.xml");
+    let result = parse_rss(&buf);
+    match result {
+        Err(_) => assert!(false, "parse rss did not return ok"),
+        _ => (),
+    }
+}
+
+#[test]
+fn is_not_ok() {
+    let buf = PathBuf::from("./tests/assets/test_not_real_rss.xml");
+    let result = parse_rss(&buf);
+    match result {
+        Ok(_) => assert!(false, "parse rss should not return ok"),
+        _ => (),
+    }
+}
+
+#[test]
+fn check_all_episodes() {
+    let pod: Podcast = make_me_podcast();
+    pod.episodes.iter().for_each(|ep| {
+        assert!(ep.title != ferris_cast_core::BAD_STRING);
+        assert!(ep.description != ferris_cast_core::BAD_STRING);
+        assert!(ep.pub_date.is_ok());
+        assert!(ep.enclosure != ferris_cast_core::BAD_STRING);
+    })
+}
+
+#[allow(dead_code)]
+fn make_me_podcast() -> Podcast {
+    let buf = PathBuf::from("./tests/assets/test_rss.xml");
+    let result = parse_rss(&buf);
+    result.expect("Failed to open xml for test")
+}
+
+#[tokio::test]
+async fn test_get_rss() -> Result<(), Box<dyn std::error::Error>> {
+    let shapiro_link: String = String::from("https://feeds.megaphone.fm/WWO8086402096");
```

Comment:
This link is now bad

Reply:
Recommend removing the test (that I admittedly wrote) because we should not be testing the fact that tokio works with a particular link.

Reply:
Well this isn't just testing whether or not tokio works with a particular link, this is testing that it downloads and parses properly from raw RSS data? Perhaps we should have it load from the local file then?

Reply:
I wouldn't recommend testing the reqwest library either which is where the downloading happens. 
Other than that we use rss:channel::read_from, but we already deal with parsing RSS in other tests.

Reply:
I now have it simply loading from a local RSS file.